scripts/west_commands/build.py

```diff
@@ -8,11 +8,14 @@
 import subprocess
 
 from west import log
-from west import cmake
 from west.configuration import config
-from west.build import DEFAULT_CMAKE_GENERATOR, is_zephyr_build
+from cmake import DEFAULT_CMAKE_GENERATOR, run_cmake, run_build, CMakeCache
+from build_helpers import is_zephyr_build, find_build_dir, BUILD_DIR_DESCRIPTION
```

Comment:
80 columns please

Reply:
It is 80? Exactly 80.

Reply:
My editor's flake8 complained. I'm going to do some other linting here so no worries.